### PR TITLE
Updated Portuguese automation

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR supports issue #467.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: .github/workflows/docgenerator.yml [hunk 1]
```diff
@@ -62,6 +62,29 @@ jobs:
         name: OWASP_MASVS-SNAPSHOT-ptbr.mobi
         path: OWASP_MASVS-SNAPSHOT-ptbr.mobi
 
+    - name: Generate Brazilian Portuguese PDF
```
Comment (cpholguera) [inline .github/workflows/docgenerator.yml:65]:
```suggestion
    - name: Generate Portugal Portuguese PDF
```

File: .github/workflows/docgenerator.yml [hunk 2]
```diff
@@ -62,6 +62,29 @@ jobs:
...
+      run: docker run --rm -u `id -u`:`id -g` -v ${PWD}:/pandoc owasp/masvs-docgenerator:0.2 '/pandoc_makedocs.sh Document-ptpt SNAPSHOT'
+    - name: Upload ptpt PDF
+      uses: actions/upload-artifact@v1
+      with:
+        name: OWASP_MASVS-SNAPSHOT-ptpt.pdf
+        path: OWASP_MASVS-SNAPSHOT-ptpt.pdf
+    - name: Upload Brazilian Portuguese EPUB
```
Comment (cpholguera) [inline .github/workflows/docgenerator.yml:72]:
```suggestion
    - name: Upload Portugal Portuguese EPUB
```

File: .github/workflows/docgenerator.yml [hunk 3]
```diff
@@ -62,6 +62,29 @@ jobs:
...
+      uses: actions/upload-artifact@v1
+      with:
+        name: OWASP_MASVS-SNAPSHOT-ptpt.epub
+        path: OWASP_MASVS-SNAPSHOT-ptpt.epub
+    - name: Upload Brazilian Portuguese DOCX
```
Comment (cpholguera) [inline .github/workflows/docgenerator.yml:77]:
```suggestion
    - name: Upload Portugal Portuguese DOCX
```

File: .github/workflows/docgenerator.yml [hunk 4]
```diff
@@ -62,6 +62,29 @@ jobs:
...
+      uses: actions/upload-artifact@v1
+      with:
+        name: OWASP_MASVS-SNAPSHOT-ptpt_WIP_.docx
+        path: OWASP_MASVS-SNAPSHOT-ptpt_WIP_.docx
+    - name: Upload Brazilian Portuguese MOBI
```
Comment (cpholguera) [inline .github/workflows/docgenerator.yml:82]:
```suggestion
    - name: Upload Portugal Portuguese MOBI
```

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -28,6 +28,8 @@ jobs:
         run: docker run -v ${PWD}:/documents owasp/masvs-docgenerator:0.2 'cd /documents/tools && python3 export.py --format xml --lang en > OWASP_MASVS-${{ steps.get_version.outputs.VERSION }}.xml'
       - name: Generate Brazilian Portuguese PDF
         run: docker run --rm -u `id -u`:`id -g` -v ${PWD}:/pandoc owasp/masvs-docgenerator:0.2 '/pandoc_makedocs.sh Document-ptbr ${{ steps.get_version.outputs.VERSION }}'
+      - name: Generate Portuguese Portuguese PDF
```
Comment (cpholguera) [inline .github/workflows/release.yml:31]:
```suggestion
      - name: Generate Portugal Portuguese PDF
```

File: Document-ptpt/LANGUAGE-METADATA [hunk 6]
```diff
@@ -1,3 +1,3 @@
-export LANGUAGE="en"
+export LANGUAGE="ptpt"
 export LANGUAGETEXT=""
 export TOC_TITLE="Table of Contents"
```
Comment (cpholguera) [inline Document-ptpt/LANGUAGE-METADATA:3]: We need the proper translation here.

Comment (commjoen) [reply Document-ptpt/LANGUAGE-METADATA:3]: Will be done later i guess :).

**Pull request merged**